node/src/components/small_network/tests.rs

```diff
@@ -143,7 +143,7 @@ impl Reactor for TestReactor {
             Event::NetworkRequest(req) => self.dispatch_event(
                 effect_builder,
                 rng,
-                Event::SmallNet(SmallNetworkEvent::from(req)),
+                Event::SmallNet(SmallNetworkEvent::NetworkRequest { req: Box::new(req) }),
```

Comment:
The `From` impl should be updated instead, if actually boxing here. We do not want to manually instantiate events if we can help it.

However, is this some sort of weird edge case? The `NetworkRequest` itself is wrapped using `Event::SmallNet` in what is essentially a newtype (it has only one field). Is `NetworkRequest` so big that this extra wrapping causes it to go over the maximum event size?

It would be more idiomatic to not box the request here.

Reply:
Added a specific `From` implementation